int8 robot
int8 current_task
time stamp

float32[] motivation
int8[] sensory_feedback
int8[] activity_suppression
float32[] impatience
int8[] impatience_reset
int8[] acquiescence
float32 pose_x
float32 pose_y
